etc/kayobe/environments/ci-aio/automated-setup.sh

```diff
@@ -2,39 +2,42 @@
 
 set -eux
 
+if [[ ! -f $BASE_PATH/vault-pw ]]; then
+    echo "Vault password file not found at $BASE_PATH/vault-pw"
+    exit 1
+fi
+
 cat << EOF | sudo tee -a /etc/hosts
 10.205.3.187 pulp-server pulp-server.internal.sms-cloud
 EOF
 
-if sudo vgdisplay | grep -q lvm2; then
-   sudo pvresize $(sudo pvs --noheadings | head -n 1 | awk '{print $1}')
-   sudo lvextend -L 4G /dev/rootvg/lv_home -r || true
-   sudo lvextend -L 4G /dev/rootvg/lv_tmp -r || true
-fi
-
 BASE_PATH=~
 KAYOBE_BRANCH=stackhpc/yoga
 KAYOBE_CONFIG_BRANCH=stackhpc/yoga
```

Comment:
Move to the top, since they may require modification?

Reply:
Done